doc/releases/release-notes-2.5.rst

```diff
@@ -376,12 +423,139 @@ Drivers and Sensors
 
 * WiFi
 
-  * Added uart bus interface for eswifi driver.
+  * eswifi:
+
+    * Added uart bus interface. This enables all Inventek modules with
+      IWIN AT Commands firmware.
+
+  * esp32:
```

Comment:
`s/esp32/esp/` This is a driver for both ESP32 and ESP8266 (and some variants of those).